src/modules/ens/utils.ts

```diff
@@ -8,8 +8,8 @@ import { createEth } from 'decentraland-dapps/dist/lib/eth'
 
 export const PRICE_IN_WEI = 100000000000000000000 // 100 MANA
 export const PRICE = fromWei(PRICE_IN_WEI.toString(), 'ether')
-
 export const MAX_NAME_SIZE = 15
+export const NAME_MIN_CHARACTERS = 1
```

Comment:
I think that we can use `MIN_NAME_SIZE = 2` just to follow the name of the variable above.